backend/src/core/core_service_impl.h

```diff
@@ -50,6 +50,7 @@ class CoreServiceImpl final : public dronecode_sdk::rpc::core::CoreService::Serv
                                        "gimbal",
                                        "info",
                                        "mission",
+                                       "offboard",
```

Comment:
You need to bump the size of the array to 10.
@JonasVautherin we might as well change this to a `std::vector` with initializer list.